halide/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+# This file differs from ../CMakeLists.txt only in the target_name_mod
+# var and lack of add_subdirectory call.
+
+if(BUILD_halide)
+
+    #add_subdirectory(test)
+
+    # Target name modification
```

Comment:
The argument for keeping this is that I easily allows defining a separate CMake script in a subfolder, e.g. for tests.